yarn.lock

```diff
@@ -3216,6 +3216,11 @@ fast-deep-equal@^2.0.1:
   resolved "https://registry.yarnpkg.com/fast-deep-equal/-/fast-deep-equal-2.0.1.tgz#7b05218ddf9667bf7f370bf7fdb2cb15fdd0aa49"
   integrity sha1-ewUhjd+WZ79/Nwv3/bLLFf3Qqkk=
 
+fast-deep-equal@^3.1.1:
```

Comment:
Would be cool if we could at some point use fast-deep-equal 3 for all our packages where we currently have a fast-deep-equal 2 dependency

Reply:
https://github.com/iov-one/iov-core/issues/1337